src/tester.tsx

```diff
@@ -111,30 +118,34 @@ class Tester {
     return this.component.text();
   }
 
-  find (selector) {
+  find (selector: string) {
     return this.wrapper.find(selector);
   }
 
   update () {
     return this.wrapper.update();
   }
 
-  async sleep (ms) {
+  async sleep (ms?: number) {
     await sleep(ms);
```

Comment:
I shouldn't be optional if we're passing it ? Or it's fine since there's a default in `sleep()` ?

Reply:
Fine since it's optional in sleep. `ms?: number` == `ms: number | undefined`